frontend/js/src/common/brainzplayer/SpotifyPlayer.tsx

```diff
@@ -617,6 +617,14 @@ export default class SpotifyPlayer
     if (!show) {
       return null;
     }
-    return <div data-testid="spotify-player">{this.getAlbumArt()}</div>;
+    return (
+      <BrainzPlayerContext.Consumer>
+        {(context) => {
+          const { volume } = context;
+          this.spotifyPlayer?.setVolume(volume / 100);
+          return <div data-testid="spotify-player">{this.getAlbumArt()}</div>;
+        }}
+      </BrainzPlayerContext.Consumer>
+    );
```

Comment:
How about instead of creating a consumer for the context, we pass down the volume as props?

It would simplify things and also prevent re-rendering whenever the context values are updated

Reply:
This improvement can be made for all the players.